scripts/py_matter_idl/matter/idl/tests/outputs/large_lighting_app/cpp-app/ember-cluster-config/AdministratorCommissioning.h

```diff
@@ -0,0 +1,33 @@
+// DO NOT EDIT - Generated file
+//
+// Application configuration for AdministratorCommissioning based on EMBER configuration
+// from inputs/large_lighting_app.matter
+#pragma once
+
+#include <app-common/zap-generated/cluster-enums.h>
+#include <app/util/cluster-config.h>
+
+#include <array>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace AdministratorCommissioning {
+namespace EmberApplicationConfig {
+
+using FeatureBitmapType = uint32_t;
```

Comment:
I'm a little confused by this, because AdministratorCommissioning in fact has a defined feature map.  Why is this ending up as uint32_t?

Reply:
I see no feature bitmap in https://github.com/project-chip/connectedhomeip/blob/master/scripts/py_matter_idl/matter/idl/tests/inputs/large_all_clusters_app.matter#L1509